Jn9e9/issue39

jn9e9 opened the pull request:
Fixes #39

Signed-off-by: John 9e9 <jn9e9e9@gmail.com>

CI test not working - not broken in this issue, as described in #40

ionut-arm commented:
Hey! Thanks for the PR, I think the build is failing because we added a new provider to Parsec which needs a submodule to be initialised, so you need to do a `git submodule update --init` before [building the service](https://github.com/parallaxsecond/parsec-client-go/blob/main/e2etest/provider_cfg/all/Dockerfile#L21). You also need to include `direct-authenticator` as a feature in that build command. I'll review the actual PR now.

ionut-arm reviewed (approved):
Looks good, thanks for the update/cleanup!

jn9e9 commented:
> 
> 
> Hey! Thanks for the PR, I think the build is failing because we added a new provider to Parsec which needs a submodule to be initialised, so you need to do a `git submodule update --init` before [building the service](https://github.com/parallaxsecond/parsec-client-go/blob/main/e2etest/provider_cfg/all/Dockerfile#L21). You also need to include `direct-authenticator` as a feature in that build command. I'll review the actual PR now.
> 
> I'll cross-post this in #40 , do you want to fix it in a separate PR or should I wait for a change here?

fixed in new pr #42, ta
